source/app/pages/documents/view.js

```diff
@@ -322,14 +288,20 @@ function Document({ currentPageNumber, dispatch, id, document, pageTitle, search
             {track === 'redaction' &&
             document.redactions &&
             Object.keys(document.redactions).length ? (
-              <div className={css.downloadButtons}>
-                <Button inverted onClick={clearReds}>
-                  Clear Redaction
-                </Button>
-                <Button className={css.downloadRedacted} onClick={downloadRedacted}>
-                ⬇ Redacted Doc
-                </Button>
-              </div>
+                <div className={css.downloadButtons}>
+                  <Button inverted onClick={clearReds}>
+                    Clear Redaction
+                  </Button>
+                  <Menu>
+                    <MenuButton as={Button}>
+                      ⬇ Redacted Doc
+                    </MenuButton>
+                    <MenuList>
+                      <MenuItem bg="inherit" border="none" fontSize="1rem" onClick={() => downloadRedacted(ExportTypes.PDF)}>PDF</MenuItem>
+                      <MenuItem bg="inherit" border="none" fontSize="1rem" onClick={() => downloadRedacted(ExportTypes.PNG)}>PNG</MenuItem>
```

Comment:
I am wondering if it makes sense to download multi page PDFs as one large PNG/ multiple PNG files. I tried the download for a 5 page PDF and it was super long. Thoughts?

Reply:
As discussed offline, we will leave as is for now